docs/source/contributing/releasing.adoc

```diff
@@ -27,8 +21,39 @@ defined in your environment as _GITHUB_ACCESS_TOKEN_.
   For example, issues with a merged pull requests should be labeled as *resolution/done*.
 . Close milestone.
 
+[[auto-relase]]
+== Automated Release
+
+An automated release can be performed as:
+
+----
+$ API_KEY=<api-key> RELEASE_VERSION=<version> make ci_release
+----
+
+where
+
+  - `api-key` : Api Key of the `minishift` ci user
+  - `version` : Expected release version. Eg : 1.0.0 (without 'v')
+
+In case, you are not able to trigger the job via command line, try running link:https://ci.centos.org/job/minishift-release[release build] by passing required `RELEASE_VERSION` and `MILESTONE_ID` information.
```

Comment:
What do you mean here? How do I trigger it in this case? curl? Why would that make a difference to running via make?